src/component/FetchItems.js

```diff
@@ -5,6 +5,7 @@ import Navbar from './Navbar';
 import DeleteToken from './DeleteToken';
 import calculateNewPurchaseValues from '../calculations';
 import dayjs from 'dayjs';
+import SortedLists from './SortedLists';
```

Comment:
Could delete FirestoreCollection import since no longer using - to clear console warning.

Reply:
Good call